examples/flow.cpp

```diff
@@ -251,15 +251,31 @@ try
     const PhaseUsage pu = Opm::phaseUsageFromDeck(deck);
     Opm::BlackoilOutputWriter outputWriter(grid, param, eclipseState, pu );
 
+    int numCells = Opm::UgGridHelpers::numCells(grid);
```

Comment:
I always forget that this beast might null (and for a reason). Having
an abstraction would make sense, then.